card-auth-server-dao/src/main/java/com/razorpay/ffs/cas/dao/enums/TransactionStatus.java

```diff
@@ -1,7 +1,10 @@
 package com.razorpay.ffs.cas.dao.enums;
 
+import lombok.Getter;
+
+@Getter
 public enum TransactionStatus {
-    CREATED("U", "Created"), // todo check created should ne mapped to N/U
+    CREATED("U", "Created"),
```

Comment:
since todo is removed, seems like we confirmed this :)
How did we do this?

Reply:
https://razorpay.slack.com/archives/C0580DVQN06/p1690185016355059?thread_ts=1690184106.040309&cid=C0580DVQN06

Discussed with Piyush and Ashish